woocommerce/woocommerce
# Add partial verification support for HPOS CLI

### All Submissions:






### Changes proposed in this Pull Request:



We already have a `--start-from` support in `wp wc cot verify_cot_data` command, but there is no `--end-at` equivalent. This PR adds the `--end-at` support as well, which allows doing partial verification as well and the user won't have to wait for migration to complete.



### How to test the changes in this Pull Request:




With HPOS enabled (or not):

1. Run the CLI command for verification  `wp wc cot verify_cot_data --start-at=xx --end-at=yy` and check that command run successfully.



### Other information:




### FOR PR REVIEWER ONLY:
